content/blog/2018-09-04-birds-taxo-traits.md

```diff
@@ -0,0 +1,559 @@
+  dplyr::mutate(habitat = dplyr::case_when(stringr::str_detect(habitat_level_1,
+                                                               "Marine") ~ "Marine",
+                                           stringr::str_detect(habitat_level_1,
+                                                               "Rocky") ~ "Rocky",
+                                           TRUE  ~ habitat_level_1)) %>%
+  ggplot() +
+  geom_bar(aes(occurrence, fill = habitat),
+           position = "fill") +
+  # palette recommended in https://github.com/clauswilke/colorblindr
+  # for people with color-vision deficiency
+  colorblindr::scale_fill_OkabeIto() +
+  theme(legend.position = "bottom") +
+  hrbrthemes::theme_ipsum(base_size = 12, 
+                          axis_title_size = 12,
+                          axis_text_size = 12) +
+  ggtitle("Habitats of birds from the county of Constance",
+          subtitle = "For birds with habitats data only for the breeding and non-breeding seasons") +
+  ylab("Proportion") +
+  xlab("Season of occurrence")
+```
+
+![](/img/blog-images/2018-09-04-birds-taxo-traits/unnamed-chunk-9-1.png)
+
+It seems that the birds present in the County of Constance might have
+different distributions depending on the breeding/non-breeding season.
+It’d be interesting to look at in the occurrence data, since `auk`
+allows zero-filling.
+
+Let’s have a look at threats data.
+
+``` r
+threats <- purrr::map_df(species_info, "threats")
+str(threats)
+```
+
+    ## 'data.frame':    414 obs. of  8 variables:
+    ##  $ id      : int  22696993 22696993 22696993 22696993 22696993 22696993 22696993 22696993 103888106 103888106 ...
+    ##  $ threat1 : chr  "Agriculture & aquaculture" "Biological resource use" "Biological resource use" "Biological resource use" ...
+    ##  $ threat2 : chr  "Annual & perennial non-timber crops" "Hunting & trapping terrestrial animals" "Hunting & trapping terrestrial animals" "Logging & wood harvesting" ...
+    ##  $ stresses: chr  "Ecosystem degradation, Ecosystem conversion" "Species mortality" "Species mortality" "Species disturbance, Ecosystem degradation" ...
+    ##  $ timing  : chr  "Agriculture & aquaculture" "Biological resource use" "Biological resource use" "Biological resource use" ...
+    ##  $ scope   : chr  "Annual & perennial non-timber crops" "Hunting & trapping terrestrial animals" "Hunting & trapping terrestrial animals" "Logging & wood harvesting" ...
+    ##  $ severity: chr  "Ongoing" "Ongoing" "Past, Likely to Return" "Ongoing" ...
+    ##  $ impact  : chr  "Ongoing" "Ongoing" "Past, Likely to Return" "Ongoing" ...
+
+As you can see, it is a depressing dataset since most threats are
+human-made, but such information can help conservation! Instead of
+diving into the fate of particular species, let’s get a general picture.
+There are 211 species, for 68 we get an entry in the threats data. What
+are the most common threats for them, now and in the future?
```

Comment:
I got slowed down trying to interpret headings threat2 and thread1. If they are now and in the future could you write "now (THREAT2) and in the future (THREAT1)" or similar?

Reply:
I'm renaming the headers